lib/statsd/statsdAggregator.js

```diff
@@ -27,16 +27,26 @@ StatsdAggregator.prototype._timeoutMetric = function(metric) {
     var _this = this;
 
     setTimeout(function() {
-        // fire increment
-        _this.statdsClient.increment(metric.metric, metric.value);
+        // This condition is because StatsD client for 0 delta fallbacks to increment by 1
+        if(metric.value) {
+            // fire increment
+            _this.statdsClient.increment(metric.metric, metric.value);
+        }
         _this.metrics = _.without(_this.metrics, metric);
     }, this.timeout);
 };
 
-StatsdAggregator.prototype.increment = function(metricName) {
+StatsdAggregator.prototype.increment = function(metricName, delta) {
     var metric = this._getMetric(metricName);
 
-    metric.value++;
+    if(typeof(delta) === "undefined") {
+        // Default delta value is increment by 1
+        metric.value++;
```

Comment:
You can use `++metric.value` instead of `metric.value++`
That syntax not use a temporaly variable.